packages/@webex/plugin-meetings/test/unit/spec/locus-info/index.js

```diff
@@ -1884,6 +1884,27 @@ describe('plugin-meetings', () => {
         });
       });
 
+      it('applyLocusDeltaData handles LOCUS_URL_CHANGED action correctly', () => {
+        const {LOCUS_URL_CHANGED} = LocusDeltaParser.loci;
+        const fakeDeltaLocus = {id: 'fake delta locus'};
+        const meeting = {
+          meetingRequest: {
+            getLocusDTO: sandbox.stub().resolves({body: fakeDeltaLocus}),
+          },
+          locusInfo: {
+            handleLocusDelta: sandbox.stub(),
+          },
+          locusUrl: 'oldLocusUrl',
```

Comment:
nitpick: the value "oldLocusUrl" is slightly confusing, because for the LOCUS_URL_CHANGED case we actually want to do the sync with a new locus url not the old one, but the new url is set somewhere else - in processDeltaEvent() so it's not part of this unit test. So the code is fine, but maybe a value "current locus url" or something like that would be better?